app/src/lib/components/BranchDropzone.svelte

```diff
@@ -31,49 +31,45 @@
 	}
 </script>
 
-<div
-	class="canvas-dropzone"
-	use:dropzone={{
-		active: 'new-dz-active',
-		hover: 'new-dz-hover',
-		onDrop,
-		accepts
-	}}
->
-	<div id="new-branch-dz" class="new-virtual-branch">
-		<div class="new-virtual-branch__content">
-			<div class="stimg">
-				<div class="stimg__hand">
-					{@html handSvg}
+<div class="canvas-dropzone">
+	<Dropzone {accepts} ondrop={onDrop}>
+		{#snippet overlay({ hovered, activated })}
+			<div class="new-virtual-branch" class:activated class:hovered>
+				<div class="new-virtual-branch__content">
+					<div class="stimg">
+						<div class="stimg__hand">
+							{@html handSvg}
+						</div>
+						<div class="stimg__top-sheet">
```

Comment:
Because this is quite a specialised dropzone, I chose not to have some generic component for the overlay that it uses.